packages/EasyWebhook/src/Stores/DoctrineDbalStore.php

```diff
@@ -82,9 +89,14 @@ public function generateWebhookId(): string
         return $this->random->uuidV4();
     }
 
+    public function getTimezone(): ?string
```

Comment:
@evannamon is this method only for the tests? If yes, then please remove it as any public method becomes something we have to maintain as people can use it.
Your main test `testStoreWithTimezone` is IMO already doing everything we need to do.